python/typelibrary.py

```diff
@@ -183,6 +183,8 @@ def alternate_names(self) -> List[str]:
 
 	def add_alternate_name(self, name: str) -> None:
 		"""Adds an extra name to this type library used during library lookups and dependency resolution"""
+		if type(name) != str:
```

Comment:
This is not idiomatic python.

Please change to 
`if not isinstance(name, str):`

Reply:
Changed that, my bad